core/packages/contracts/foundry.toml

```diff
@@ -0,0 +1,16 @@
+[profile.default]
+solc = '0.8.9'
```

Comment:
Isn't this [solc_version](https://book.getfoundry.sh/reference/config/solidity-compiler#solc_version)?

Reply:
both works, but seems [solc_version](https://book.getfoundry.sh/reference/config/solidity-compiler#solc_version) is more recently so just change to it